src/test/java/com/iexec/core/task/TaskServiceTests.java

```diff
@@ -38,7 +37,8 @@
     private final static String DAPP_NAME = "dappName";
     private final static String COMMAND_LINE = "commandLine";
     private final long maxExecutionTime = 60000;
-    private final String noSgxTag = "0x0";
+    private final static String NO_SGX_TAG = "0x0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
We already have this value here https://github.com/iExecBlockchainComputing/iexec-common/blob/002ed4651e87983e375f3f5ceff13657342b8177/src/main/java/com/iexec/common/utils/BytesUtils.java#L10

Reply:
changed.